core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -52,7 +58,8 @@ oppia.directive('questionsTab', [
               return;
             }
             EditableQuestionBackendApiService.createQuestion(
-              $scope.skillId, $scope.question.toBackendDict(true)
+              $scope.skillId, $scope.question.toBackendDict(
+                true)
```

Comment:
Why this change? If you can fit an arg entirely on one line, do so.

Reply:
Done